resources/js/Pages/Landing/SearchPanel.vue

```diff
@@ -11,6 +14,20 @@ const props = defineProps({
   countries: Array,
 })
 
+const authenticated = ref(false)
+const user = ref(null)
+
+onMounted(() => {
+  axios.get('/user', { timeout: 5000 })
+    .then(response => {
+      user.value = response.data
+      authenticated.value = true
+    })
+    .catch(() => {
+      authenticated.value = false
+    })
+})
+
```

Comment:
Can be replaced with:
```suggestion
const authenticated = computed(() => usePage().props.auth.isAuth)
```
After import:
`import { usePage } from "@inertiajs/vue3"
`

Reply:
Replaced